spec/index.bs

```diff
@@ -765,11 +765,11 @@ Device {#device}
 
 <script type=idl>
 dictionary GPUExtensions {
-    boolean anisotropicFiltering;
+    boolean anisotropicFiltering = false;
 };
 
 dictionary GPULimits {
-    u32 maxBindGroups;
+    u32 maxBindGroups = 4;
```

Comment:
Again, this doesn't make sense to me because it should never be encountered by the user. It's a read-only dictionary provided by the API.

Reply:
It's also passed in via GPUDeviceDescriptor.

Reply:
ah, nice spot. Missed that. Sorry for the noise